### ISSUE TITLE
Error on new application if social profiles are disabled

### ISSUE THREAD

**Issue by cjriks** [cross-solution/YAWIK#218]
If the owner of a job has unchecked 'social profiles' in the application settings (/de/settings/Applications) the system gives an error on calling the apply page (/de/apply/1234567890...):

```
2016-06-01 15:45:12 EXCEPTION No element by the name of [profiles] found in form in /usr/local/YAWIK/vendor/zendframework/zend-form/src/Fieldset.php on line 218
[Trace]
        File: /usr/local/YAWIK/module/Core/src/Core/Form/Container.php
        Line: 630
    Function: get
       Class: Zend\Form\Fieldset
        Type: method
------------------------------
        File: /usr/local/YAWIK/module/Applications/src/Applications/Controller/ApplyController.php
        Line: 199
    Function: get
       Class: Core\Form\Container
        Type: method
------------------------------
        File: /usr/local/YAWIK/vendor/zendframework/zend-mvc/src/Controller/AbstractActionController.php
        Line: 82
    Function: indexAction
       Class: Applications\Controller\ApplyController
        Type: method
------------------------------
    Function: onDispatch
       Class: Zend\Mvc\Controller\AbstractActionController
        Type: method
------------------------------
        File: /usr/local/YAWIK/vendor/zendframework/zend-eventmanager/src/EventManager.php
        Line: 490
    Function: call_user_func
------------------------------
        File: /usr/local/YAWIK/vendor/zendframework/zend-eventmanager/src/EventManager.php
        Line: 214
    Function: triggerListeners
       Class: Zend\EventManager\EventManager
        Type: method
------------------------------
        File: /usr/local/YAWIK/vendor/zendframework/zend-mvc/src/Controller/AbstractController.php
        Line: 118
    Function: trigger
       Class: Zend\EventManager\EventManager
        Type: method
------------------------------
        File: /usr/local/YAWIK/vendor/zendframework/zend-mvc/src/DispatchListener.php
        Line: 93
    Function: dispatch
       Class: Zend\Mvc\Controller\AbstractController
        Type: method
------------------------------
    Function: onDispatch
       Class: Zend\Mvc\DispatchListener
        Type: method
------------------------------
        File: /usr/local/YAWIK/vendor/zendframework/zend-eventmanager/src/EventManager.php
        Line: 490
    Function: call_user_func
------------------------------
        File: /usr/local/YAWIK/vendor/zendframework/zend-eventmanager/src/EventManager.php
        Line: 214
    Function: triggerListeners
       Class: Zend\EventManager\EventManager
        Type: method
------------------------------
        File: /usr/local/YAWIK/vendor/zendframework/zend-mvc/src/Application.php
        Line: 314
    Function: trigger
       Class: Zend\EventManager\EventManager
        Type: method
------------------------------
        File: /usr/local/YAWIK/public/index.php
        Line: 49
    Function: run
       Class: Zend\Mvc\Application
        Type: method
------------------------------
```

**Comment by cbleek** [comment]
This has beed fixed in the develop, I guess.

**Comment by TiSiE** [comment]
Yes, fixed with #215 during the release process of 0.25.
